charts/apisix/templates/configmap.yaml

```diff
@@ -276,12 +276,21 @@ data:
               role: viewer
         etcd:
         {{- if .Values.etcd.enabled }}
+          {{- if .Values.etcd.auth.tls.enabled }}
+          host:                                 # it's possible to define multiple etcd hosts addresses of the same etcd cluster.
+            {{- if .Values.etcd.fullnameOverride }}
+            - "https://{{ .Values.etcd.fullnameOverride }}:{{ .Values.etcd.service.port }}"
```

Comment:
Is it possible to define a `scheme` variable here (see https://helm.sh/docs/chart_template_guide/variables/), so that we can reduce the template complexity?

Reply:
yes

Reply:
How about a helper like:
```
{{/*
Scheme to use while connecting etcd
*/}}
{{- define "apisix.etcd.auth.scheme" -}}
{{- if .Values.etcd.auth.tls.enabled }}
{{- "https" }}
{{- else }}
{{- "http" }}
{{- end }}
{{- end }}
```

Reply:
Cool.